src/main/java/com/minecolonies/colony/buildings/BuildingHome.java

```diff
@@ -140,7 +146,7 @@ public void readFromNBT(NBTTagCompound compound)
             CitizenData citizen = getColony().getCitizen(citizenId);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'citizen' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)